app/views/layouts/api_guidance.html.erb

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en" class="govuk-template admin--wide">
+<%= render partial: "layouts/shared/head" %>
+
+<body class="govuk-template__body ">
+
+<a href="#main-content" class="govuk-skip-link">Skip to main content</a>
+
+<%= render partial: "layouts/shared/header" %>
+
+<div class="govuk-width-container">
```

Comment:
If you branch off #1222 you could pull in the new navigation components here

Reply:
Yeah that's fair, I don't think I called it out explicitly in any of the tickets but I've added it to the index page ticket 👍